core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -23,15 +23,16 @@
  * through.
  */
 export type SectionIdent =
-  'sect' |
```

Comment:
I'd put this in file order, which is alpha with the exception of `sect` which is always first.